plugins/aladino/functions/selectFromJSON.go

```
@@ -0,0 +1,57 @@
+// Copyright (C) 2022 Explore.dev, Unipessoal Lda - All Rights Reserved
+// Use of this source code is governed by a license that can be
+// found in the LICENSE file.
+
+package plugins_aladino_functions
+
+import (
+	"encoding/json"
+	"log"
+
+	"github.com/ohler55/ojg/jp"
+	"github.com/reviewpad/reviewpad/v3/handler"
+	"github.com/reviewpad/reviewpad/v3/lang/aladino"
+)
+
+func SelectFromJSON() *aladino.BuiltInFunction {
+	return &aladino.BuiltInFunction{
+		Type:           aladino.BuildFunctionType([]aladino.Type{aladino.BuildJSONType(), aladino.BuildStringType()}, aladino.BuildStringType()),
+		Code:           selectFromJSONCode,
+		SupportedKinds: []handler.TargetEntityKind{handler.PullRequest, handler.Issue},
+	}
+}
+
+func selectFromJSONCode(e aladino.Env, args []aladino.Value) (aladino.Value, error) {
+	jsonValue := args[0].(*aladino.JSONValue).Val
+	expr := args[1].(*aladino.StringValue).Val
+
+	parsedExpression, err := jp.ParseString(expr)
+	if err != nil {
+		return nil, err
+	}
+
+	results := parsedExpression.Get(jsonValue)
+
+	if len(results) == 0 {
+		log.Printf(`selectFromJSON: nothing found at path "%s"\n`, expr)
+		return aladino.BuildStringValue(""), nil
```

Shouldn't we return an error here? How will we differentiate between returning "" and an error?

---

this one was changed due to the suggestion here https://github.com/reviewpad/reviewpad/pull/563#discussion_r1054611095, what do you think?

---

okay, thanks for pointing that out - this just need to be documented very well in the docs :)